modules/BuildTools.Tasks/BuildTools.Tasks.csproj

```diff
@@ -9,16 +9,15 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\..\shared\Microsoft.Extensions.CommandLineUtils.Sources\**\*.cs" />
```

Comment:
The hypothetical scenario is that there is no feed at all. If we don't have any dotnet.myget.org feed available, we're stuck.

FWIW I wish we didn't have to copy source either, but it seems better than the alternative: git submodules.